docs/sources/alerting/fundamentals/_index.md

```diff
@@ -91,40 +95,40 @@ A rule using the PromQL expression above creates as many alert instances as the
 
 {{< figure src="/static/img/docs/alerting/unified/multi-dimensional-alert.png" caption="Multiple alert instances from a single alert rule" >}}
 
-[Alert rules are frequently evaluated](ref:alert-rule-evaluation) and the state of their alert instances is updated accordingly. Only alert instances that are in a firing or resolved state are routed to notification policies to be handled.
+[Alert rules are frequently evaluated](ref:alert-rule-evaluation) and the state of their alert instances is updated accordingly. Only alert instances that are in a firing or resolved state are sent for notifications.
 
-### Notification policies
+### Contact points
 
-[Notification policies](ref:notification-policies) group alerts and then route them to contact points. They determine when notifications are sent, and how often notifications should be repeated.
+[Contact points](ref:contact-points) determine the notification message and where notifications are sent. For example, you might have a contact point that sends notifications to an email address, to Slack, to an incident management system (IRM) such as Grafana OnCall or Pagerduty, or to a webhook.
 
-Alert instances are matched to notification policies using label matchers. This provides a flexible way to organize and route alerts to different receivers.
+In the alert rule, you can configure the contact point to receive the alert notifications or use notification policies instead.
```

Comment:
https://grafana.com/docs/grafana/next/alerting/alerting-rules/create-grafana-managed-rule/#configure-labels-and-notifications

> Configure who receives a notification when an alert rule fires by either choosing Select contact point or Use notification policy.

It's an either or.

Reply:
really?but notification policys are how and contact points are to who - so you need both..right? i mean...you could use one or the other...but in other occasions you'd use both?

Reply:
![Screenshot 2024-06-13 at 16 57 52](https://github.com/grafana/grafana/assets/825430/29699174-a0c3-43d8-9a8c-d8fee9bee4ca)


Yes, you use always "both" behind the scene. But from the user perspective, they choose either one or the other:


![Screenshot 2024-06-13 at 16 57 22](https://github.com/grafana/grafana/assets/825430/8302f7e7-de6d-4e7a-8215-ead6be41a814)


![Screenshot 2024-06-13 at 16 57 34](https://github.com/grafana/grafana/assets/825430/5fd31c1f-9f10-4f15-8299-6853a7529470)


That is why I want to simplify by saying you either:
- choose a contact point
- use notification policies for handling alerts with greater flexibility

Reply:
thats the simple use case - if theyre using email and the default contact point - but if they want to use policies and choose contact poitns and add contact points - so i think if you want to go this route - you're going to have to introduce simple vs advanced rule creation concepts

Reply:
@brendamuir, this is the intro, so I'd like not to go into details:


> In the alert rule, you can choose a contact point to receive the alert notifications or use notification policies instead.


This sentence tries to connect ContactPoint & NotificationPolicies with the `How it works` diagram, the above the fold image. See full


> ### Contact points
> [Contact points](https://grafana.com/docs/grafana/latest/alerting/fundamentals/notifications/contact-points/) determine the notification message and where notifications are sent. For example, you might have a contact point that sends notifications to an email address, to Slack, to an incident management system (IRM) such as Grafana OnCall or Pagerduty, or to a webhook.
> 
> In the alert rule, you can choose a contact point to receive the alert notifications or use notification policies instead.
> 
> ### Notification policies
> [Notification policies](https://grafana.com/docs/grafana/latest/alerting/fundamentals/notifications/notification-policies/) provide a flexible method to handle alert notifications for larger systems.
> 
> Notification policies routes alerts to contact points via label matching. Each notification policy consists of a set of label matchers (0 or more) that specify which alert instances (identified by their labels) they handle. Notification policies are defined in a tree structure, where the root of the notification policy tree is the Default notification policy, which ensures all alert instances are handled.
> 
> Each notification policy decides where to send the alert (contact point) and when to send the notification (timing options). Additionally, it can [group multiple firing alert instances into a single notification](https://grafana.com/docs/grafana/latest/alerting/fundamentals/notifications/group-alert-notifications/) to reduce alert noise.


That particular sentence is also mentioned on the Alert rule section. If you think it is better to remove it, please, lmk. 

This is the last pending issue in the review 🙏